samples/system-test/detect.v2beta1_test.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const test = require(`ava`);
+const {runAsync} = require('@google-cloud/nodejs-repo-tools');
+
+const cmd = 'node detect.v2beta1.js';
+// const sessionId = require('uuid/v1')();
+// const projectId = process.env.GCLOUD_PROJECT;
+const testQuery = [`How do I sign up?`];
+
+const testKnowledgeBaseName = 'TestKnowBase';
+
+let knowbaseDisplayName;
+let knowbaseFullName;
+
+let documentFullPath;
+const testDocName = `TestDoc`;
+const testDocumentPath = `https://dialogflow.com/docs/concepts/google-projects-faq`;
+
+test.serial(`It should create a knowledge base`, async t => {
+  console.log(`displayname should be: ${testKnowledgeBaseName}`);
+  const output = await runAsync(
+    `${cmd} createKnowledgeBase -k ${testKnowledgeBaseName}`
+  );
+  const parsedOut = output.split(`\n`);
+  console.log(`parsedOut[0] is: ${parsedOut[0]}`);
+  console.log(`parsedOut[1] is: ${parsedOut[1]}`);
+  const confirmName = parsedOut[1].split(`:`)[1].trim();
+  console.log(`Confirm Name is: ${confirmName}`);
+  t.true(confirmName === testKnowledgeBaseName);
+  knowbaseDisplayName = confirmName;
+  knowbaseFullName = parsedOut[0].split(`:`)[1].trim();
+  console.log(`Full Name is: ${knowbaseFullName}`);
+  console.log(`Output is: ${output}`);
+});
+
+test.serial(
+  `It should register newly created knowledge base with getKnowledgeBase`,
+  async t => {
+    const output = await runAsync(
+      `${cmd} getKnowledgeBase -n "${knowbaseFullName}"`
+    );
+    console.log(`getKnowledgeBase: input is ${knowbaseFullName}`);
+    const parsedOut = output.split(`\n`);
+    console.log(`getKnowledgeBase: output is ${output}`);
+    t.true(
+      parsedOut[0].includes(knowbaseFullName) &&
+        parsedOut[1].includes(knowbaseDisplayName)
+    );
+  }
+);
+
+test.serial(
+  `It should locate knowledge base with listKnowledgeBases`,
+  async t => {
+    const output = await runAsync(`${cmd} listKnowledgeBases`);
+    const parsedOut = output.split(`\n`);
+    parsedOut.forEach(p => {
+      if (p.includes(testKnowledgeBaseName)) t.pass();
```

Comment:
Why isn't this using a `t.true(output.includes(thing))`? Update it to use that pattern. Simply assert that the name of present knowledgebases should show up.